src/lib/mac-address.ts

```diff
@@ -0,0 +1,20 @@
+import * as fs from 'mz/fs';
+import * as path from 'path';
+
+export async function getAll(sysClassNet: string): Promise<string | undefined> {
+	if (!(await fs.exists(sysClassNet))) {
+		return '';
```

Comment:
why not `undefined`, given it's part of the function signature?

Reply:
Oversight. Corrected.